app/assets/javascripts/search.js

```diff
@@ -7,12 +7,14 @@
 
   var search = {
     init: function () {
-      var $searchResults = $('#results .results-list');
+      if (window.location.pathname === "/search") {
```

Comment:
Instead of tying this to the path in the URL, is there some other way we could tie this to the search page.  I worry that this is brittle.

Could we not trigger this based on the presence of the `#results .results-list` element (or maybe something even more specific).